src/Caching/Storages/SQLiteJournal.php

```diff
@@ -41,10 +42,9 @@ public function __construct($path)
 				key BLOB NOT NULL,
 				priority INT NOT NULL
 			);
-			CREATE INDEX IF NOT EXISTS idx_tags_key ON tags(key);
```

Comment:
SQLite can use idx_tags_key_tag index  for searching in key column. Therefore idx_tags_key is needless.